src/main/java/uk/gov/hmcts/darts/arm/config/ArmApiConfigurationProperties.java

```diff
@@ -23,5 +23,7 @@ public class ArmApiConfigurationProperties {
     private URL url;
     @NotEmpty
     private String updateMetadataPath;
+    @NotEmpty
```

Comment:
Wondering whether we should also specify the `downloadDataPath` property here for completeness just